custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json

```diff
@@ -292,6 +292,87 @@
             "description": "Statistical models",
             "visibilityCondition": "['choose_algorithms', 'customize_algorithms'].includes(model.forecasting_style)"
         },
+        {
+            "name": "season_length_min",
+            "label": "Seasonality",
+            "description": "Length of the training data season to use in statistical models.",
```

Comment:
Not sure which is the best description. @clemencebic what do you think?

Reply:
changes here: ce70e5a